components/Tables/ContributorsTable.tsx

```diff
@@ -231,19 +233,19 @@ const mobileColumns = ({ isLoggedIn }: { isLoggedIn: boolean }) => [
   },
 ];
 
-export default function ContributorsTable({
+export default function ContributorsTable<T extends Contributor>({
   contributors,
   meta,
   oscrSorting,
   setOscrSorting,
   isLoading,
   isError,
-}: ContributorsTableProps) {
+}: ContributorsTableProps<T>) {
   const isMobile = useMediaQuery("(max-width: 640px)");
   const router = useRouter();
   const { userId } = useSupabaseAuth();
   const isLoggedIn = Boolean(userId);
-  const repository = `${router.query.org}/${router.query.repo}`;
+  const repository = router.query.org && router.query.repo ? `${router.query.org}/${router.query.repo}` : undefined;
```

Comment:
In the case of contributor insights, there is no repo, so we need to check if the org/repo name exist.